### PR TITLE
mkcloud: cleanup duplicate want_* proposal vars

### PR DESCRIPTION
**Pull request by bmwiedemann**
because people can already use e.g. want_aodh_proposal=0

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -1126,20 +1126,16 @@ Optional
...
-        Deployed by default. Set to 0 to prevent sahara from being deployed.
-    want_murano=1 (default=0)
-        It's not deployed by default. Set to 1 to deploy murano.
-    want_trove=0 (default=1)
-        Deployed by default. Set to 0 to prevent trove from being deployed.
-    want_aodh=1 (default=1)
-        Deployed by default. Set to 0 to prevent aodh from being deployed.
+    want_\$PROPOSAL_proposal=[01]
+        Set to 0 to prevent \$PROPOSAL from being deployed. e.g.
+        want_aodh_proposal=0
```
Comment (aplanas) [inline scripts/mkcloud]: I think that this as active by default

File: scripts/lib/mkcloud-common.sh [hunk 2]
```diff
@@ -22,6 +22,10 @@ fi
 : ${architectures:='aarch64 x86_64 s390x'}
 : ${nodenumberlonelynode:=0}
 : ${want_mtu_size:=1500}
+# proposals:
+: ${want_magnum_proposal:=0}
+: ${want_monasca_proposal:=0}
+: ${want_murano_proposal:=0}
```
Comment (aplanas) [inline scripts/lib/mkcloud-common.sh]: ${want_barbican_proposal:=1}
 ${want_sahara_proposal:=1}
 ${want_trove_proposal:=1}
 ${want_aodh_proposal:=1}

File: scripts/mkcloud (hunk 1 above)
Comment (jdsn) [inline scripts/mkcloud]: correct, thats why the example shows how to disable it

File: scripts/lib/mkcloud-common.sh (hunk 2 above)
Comment (jdsn) [inline scripts/lib/mkcloud-common.sh]: nope
```
${want_magnum_proposal:=0}
${want_monasca_proposal:=0}
${want_murano_proposal:=0}
```
all were off by default, see diff

Comment (bmwiedemann) [inline scripts/lib/mkcloud-common.sh]: that is implicit. we dont have want_nova_proposal=1 either.

File: scripts/mkcloud (hunk 1 above)
Comment (aplanas) [inline scripts/mkcloud]: So where I can see that is by default on and by default off?

File: scripts/lib/mkcloud-common.sh (hunk 2 above)
Comment (aplanas) [inline scripts/lib/mkcloud-common.sh]: @bmwiedemann ah true, thanks!

File: scripts/mkcloud (hunk 1 above)
Comment (jdsn) [inline scripts/mkcloud]: @aplanas read:

> Set to 0 to prevent \$PROPOSAL from being deployed. e.g.

**Comment by jdsn** [comment]
maybe the comment needs to be a bit more verbose about what is on and off by default

File: scripts/mkcloud (hunk 1 above)
Comment (aplanas) [inline scripts/mkcloud]: @jdsn I am not asking the meaning of 0 or 1, I am asking how I can see the proposal that are by default on and off. I think that this "help" needs to explain that all proposals are on by default, except magnum, monasca and murano.

Comment (jdsn) [inline scripts/mkcloud]: @aplanas 

> I am asking how I can see the proposal that are by default on and of

And I answered this question. Background: I forced Bernhard (in a prereview) to add the hint in the brackets that the following proposals are off by default).

Comment (jdsn) [inline scripts/mkcloud]: @aplanas 
What I wanted to highlight was this:
> Set to 0 to **prevent** $PROPOSAL from being deployed. e.g.

and

> Set to 1 to **enable** $PROPOSAL to be deployed (**that are default off**). e.g.

Maybe that makes it clearer.

Comment (jdsn) [inline scripts/mkcloud]: but as I already wrote, I guess this misunderstanding is a hint that we need the comments to be a bit more verbose and less implicit :)

Comment (aplanas) [inline scripts/mkcloud]: @jdsn please, understand that I am not asking to @jdsn to explain to me what are the defaults. I am very clear in my previous comment: in this `help` section I miss that is not explained what are the defaults: ON all, except the previous list.

If I am wrong, can you point to me where is explained?

Comment (aplanas) [inline scripts/mkcloud]: @jdsn yes, I would prefer a more explicit version here

Comment (jdsn) [inline scripts/mkcloud]: @aplanas 

Well it is explained, its just a bit implicit. And btw. it is not less nor more implicit than before :)

**Pull request merged**